_posts/2020-09-26-bring-your-own-datatypes.md

```diff
@@ -227,15 +229,15 @@ This can be any function
 In our case,
   we use a helper function
   provided by the Bring Your Own Datatypes framework.
-`tvm.datatype.create_lower_func('BFloat16Add')`
+`tvm.target.datatype.create_lower_func({16:'Posit16es2Add'})`
   creates a lowering function
   for the common pattern described above.
 The resulting function
   converts the arguments of the given node
   to `uint16_t`,
   and then converts the node itself
   into a call to the given function name
-  (in this case, `'BFloat16Add'`).
+  (in this case, `'Posit16es2Add'`) for `posit`s of bit length 16.
```

Comment:
I think this last bit of the sentence should be in the parentheses, i.e. (in this case, `'Posit16es2Add'` for `posit`s of bit length 16).